src/components/Votes/Votes.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { TiArrowUpOutline,
+  TiArrowUpThick,
+  TiArrowDownOutline,
+  TiArrowDownThick } from 'react-icons/ti';
+
+export function Votes({ score }) {
+  const [ voteValue, setVoteValue ] = useState('default');
+
+  const onHandleVote = (newValue) => {
+    if (newValue === voteValue) {
+      setVoteValue(0);
+    } else if (newValue === 1) {
+      setVoteValue(1);
+    } else {
+      setVoteValue(-1);
+    }
+  };
+
+  const renderUpVote = () => {
```

Comment:
Better make component which will get 2 Component Types (for default and active state) and flag of activeness and use it in up/down vote cases 

here is this component's usage example
`
<Vote
  iconType={TiArrowUpOutline}
  activeIconType={TiArrowUpThick}
  isActive={voteValue === 1}
/>
`